test/test-applications/integrations/Samples.Kafka/Properties/launchSettings.json

```diff
@@ -0,0 +1,20 @@
+{
+  "profiles": {
+    "Samples.Kafka": {
+      "commandName": "Project",
+      "environmentVariables": {
+        "DD_DOTNET_TRACER_HOME": "..\\..\\..\\..\\src\\bin\\tracer-home",
+        "COR_ENABLE_PROFILING": "1",
+        "CORECLR_ENABLE_PROFILING": "1",
+        "COR_PROFILER_PATH": "..\\..\\..\\..\\src\\bin\\tracer-home\\win-x64\\Datadog.Trace.ClrProfiler.Native.dll",
+        "DD_DUMP_ILREWRITE_ENABLED": "1",
+        "CORECLR_PROFILER": "{846F5F1C-F9AE-4B07-969E-05C26BC060D8}",
+        "DD_TRACE_CALLTARGET_ENABLED": "true",
+        "DD_INTEGRATIONS": "..\\..\\..\\..\\src\\bin\\tracer-home\\integrations.json",
+        "CORECLR_PROFILER_PATH": "..\\..\\..\\..\\src\\bin\\tracer-home\\win-x64\\Datadog.Trace.ClrProfiler.Native.dll",
+        "COR_PROFILER": "{846F5F1C-F9AE-4B07-969E-05C26BC060D8}"
```

Comment:
You can use variables here instead of relative paths. This will be helpful when we move directories around (again). Here's an example: https://github.com/DataDog/dd-trace-dotnet/blob/91196b609f3f39beabb70c72c1de1f8fc2072400/test/test-applications/integrations/Samples.AspNetCoreMvc21/Properties/launchSettings.json#L17-L26

Reply:
Also using `src/bin/tracer-home` is not entirely correct. I don't think we build the tracer home directory in CI before running tests (although we probably should do that instead of copying the files into each sample's output path).

Reply:
Yeah, I'm not convinced that works 🤔 Those are MSBuild variables, but launchsettings.json doesn't do that interpolation., so the env variables become exactly that E.g. if I run that Samples.AspNetCoreMvc21 project, I see this: 
![image](https://user-images.githubusercontent.com/18755388/118846414-9520fd00-b8c4-11eb-8f39-6f1964d54899.png)

Reply:
> Also using ...win-x64\\Datadog.Trace.ClrProfiler.Native.dll won't work when testing x86

I'll give you that one 😛 I'm inclined to just remove these anyway - they're only for local testing, and I'm inclined to having a Nuke target that sets all this up for us...

Reply:
>Those are MSBuild variables, but launchsettings.json doesn't do that interpolation

Oh, right. I think they only work when running from inside VS. It's been a long time, but I guess we decided this was better than hard-coded or relative paths.

> I'm inclined to just remove these anyway

I agree, but being able to run them from VS makes it easier to debug the attached tracer.

Reply:
Updated in https://github.com/DataDog/dd-trace-dotnet/pull/1444/commits/39873364a3ccc860177f66f81f5699069e0fe690